recipes/nuraft/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
As you are using targets, prefer `target_compile_features(${PROJECT_NAME} PUBLIC cxx_std_11)`